### PR TITLE
pyupgrade + black on `/scripts`

### PR DESCRIPTION
**Pull request by jimman2003**
Progress towards #5836 , #4776 

refactor
### Stakeholders

@cclauss , @cdrini

### CODE REVIEW

File: scripts/sitemaps/sitemap.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ def process_dump(dumpfile):
         if sortkey:
             yield [sortkey, path, last_modified]
 
-re_key = re.compile("^/(authors|works)/OL\d+[AMW]$")
+re_key = re.compile(r"^/(authors|works)/OL\d+[AMW]$")
```
Comment (cclauss) [inline scripts/sitemaps/sitemap.py:86]:
```suggestion


```

File: scripts/jsondump.py [hunk 2]
```diff
@@ -124,7 +123,7 @@ def modified(db, date):
         $ python jsondump.py modified dbname YYYY-MM-DD
     """
     import os
-    os.system("""psql %s -t -c "select key from thing where last_modified >= '%s' and last_modified < (date '%s' + interval '1 day')" """ % (db, date, date))
+    os.system(f"""psql {db} -t -c "select key from thing where last_modified >= '{date}' and last_modified < (date '{date}' + interval '1 day')" """)
```
Comment (cclauss) [inline scripts/jsondump.py:126]: We will land this PR even with this test failing.

**Comment by cdrini** [comment]
If you rebase now that https://github.com/internetarchive/openlibrary/pull/5842 is merged, it should work 🚀

**Comment by cdrini** [comment]
Rebased :+1:

**Review by cclauss — APPROVED** [review]
LGTM.  Thanks for doing this!  Next steps.
* [ ] pyupgrade + black for everything outside of `/openlibrary`
* [ ] pyupgrade + black on `/openlibrary`
* [ ] add pyupgrade and black to pre-commit

**Pull request merged**